flask_jwt_extended/view_decorators.py

```diff
@@ -35,7 +35,9 @@ def _verify_token_is_fresh(jwt_header, jwt_data):
             raise FreshTokenRequired("Fresh token required", jwt_header, jwt_data)
 
 
-def verify_jwt_in_request(optional=False, fresh=False, refresh=False, locations=None):
+def verify_jwt_in_request(
+    optional=False, fresh=False, refresh=False, locations=None, token_type="access"
+):
```

Comment:
I'm not sure I like having `refresh` and `token_type` in here, that seems confusing to understand from a top level API point of view. I'm not entirely sue how that could be handled without a breaking change though. Hmm... 🤔 🤔 🤔